Packs/Phishing/Playbooks/Phishing_-_Generic_v3.yml

```diff
@@ -3067,6 +2750,211 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "213":
+    id: "213"
+    taskid: 3236a5b0-5234-448d-8842-67142d698e5d
+    type: playbook
+    task:
+      id: 3236a5b0-5234-448d-8842-67142d698e5d
+      version: -1
+      name: Block Indicators - Generic v3
+      description: |+
+        This playbook blocks malicious Indicators using all integrations that are enabled, using the following sub-playbooks:
+
+        - Block URL - Generic v2
+        - Block Account - Generic v2
+        - Block IP - Generic v3
+        - Block File - Generic v2
+        - Block Email - Generic
+        - Block Domain - Generic
+
+      playbookName: Block Indicators - Generic v3
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "43"
+    scriptarguments:
+      AutoBlockIndicators:
+        complex:
+          root: inputs.BlockIndicators
+      AutoCommit:
+        simple: "No"
```

Comment:
Is this input have a default value?  if yes, please remove this value. In case the user wants the block indicators automatically this input will not allow them to do so ( and they might prefer handling it through the Block Indicators playbook input default value).